sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md

```diff
@@ -5,9 +5,21 @@
   - `offset` is the offset of the text from the start of the document
   
 **New features**
-- Added the support for Personally Identifiable Information(PII) entity recognition feature.
-  To use this feature, you need to make sure you are using the 
-  service's v3.1-preview.1 API.
+- Added support for Personally Identifiable Information(PII) entity recognition feature.
+  To use this feature, you need to make sure you are using the service's v3.1-preview.1 API.
+  
+  New synchronous API introduced:
```

Comment:
Do we always put this in the changelog? It seems redundant to put all the overloads instead of just saying PII functionality is included like you did above.

Reply:
i can remove the all API here.